CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 0.18.1
```

Comment:
we use [next] until we decide the next version (if its patch or minor), but you probably guessed right here haha